pkgs/by-name/fi/firewalld/specify-localedir.patch

```diff
@@ -0,0 +1,12 @@
+--- a/src/firewall/config/__init__.py.in
++++ b/src/firewall/config/__init__.py.in
+@@ -19,6 +19,9 @@
+ DOMAIN = "firewalld"
+ import gettext
+ 
++locale.bindtextdomain(DOMAIN, "/usr/share/locale")
++gettext.bindtextdomain(DOMAIN, "/usr/share/locale")
```

Comment:
For nixos that would be $out/share/local and here probably just /share/local

Reply:
The purpose is to make it easier to replace `/usr` with actual `$out`.  It's something I wish to upstream to firewalld, so it's better to just do it this way.